### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by flashbackzoo**
Required for https://github.com/silverstripe/silverstripe-cms/pull/1221

### CODE REVIEW

**Comment by tractorcow** [comment]
The icon for the "bulk actions" looks a little squashed.

image

**Comment by tractorcow** [comment]
It looks a bit better

image

With position: relative; top: 1px; it looks a bit more lined up. ;D

image

**Comment by flashbackzoo** [comment]
Weird - it doesn't look misaligned for me...

I'll add the positioning anyway :)

**Comment by tractorcow** [comment]
image

The button is 1px lower and 1px higher than the "add new" link.

**Comment by flashbackzoo** [comment]
Fixed in the CMS PR

**Comment by flashbackzoo** [comment]
Updated with @scott1702 panel styling.
filter-panel

File: templates/forms/DropdownField.ss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <select $AttributesHTML>
 <% loop $Options %>
-	<option value="$Value.XML"<% if $Selected %> selected="selected"<% end_if %><% if $Disabled %> disabled="disabled"<% end_if %>><% if Title %>$Title.XML<% else %>&nbsp;<% end_if %></option>
```
Comment (tractorcow) [inline templates/forms/DropdownField.ss]: It looks as thought this broke the test at https://travis-ci.org/silverstripe/silverstripe-framework/jobs/67920076

Comment (flashbackzoo) [inline templates/forms/DropdownField.ss]: ok - i'll take a look

Comment (flashbackzoo) [inline templates/forms/DropdownField.ss]: that test is passing for me locally...

unit-test

any ideas?

Comment (tractorcow) [inline templates/forms/DropdownField.ss]: it fails for me locally. :dancer: 

image

**Comment by tractorcow** [comment]
I've fixed the test case at https://github.com/open-sausages/silverstripe-framework/commit/99bc9877284d5cb6348fadd6a9fe15042b3ffe61

**Comment by tractorcow** [comment]
At the moment CMS behat tests are failing due to this change; Am working on a fix locally.

**Comment by tractorcow** [comment]
Behat tests won't pass until this is merged... I've tested these locally and behat is fine.

**Pull request merged**